src/time_utils.c

```diff
@@ -535,3 +536,22 @@ ts_subtract_integer_from_now_saturating(Oid now_func, int64 interval, Oid timety
 		res = nowval - interval;
 	return res;
 }
+
+/* return mock time for testing */
+Datum
+ts_get_mock_time_or_current_time(void)
```

Comment:
Can the entire function be available in Debug, like the GUC used in it? I see that the function is only used in Debug.

Reply:
done.